elastic/kibana
# [Reporting] Add deprecation titles

## Summary

Closes https://github.com/elastic/kibana/issues/113190

Also added `i18n` to messages.

## How to test

1. Start Kibana on 7.x with these changes applied
2. Navigate to upgrade assistant and confirm that expected titles are present in the Kibana section (per screenshots)

## Screenshots

<img width="1227" alt="Screenshot 2021-09-29 at 13 02 55" src="https://user-images.githubusercontent.com/8155004/135256808-967a2fc7-b50b-4b5b-87b6-9b315ce95f1a.png">

<img width="495" alt="Screenshot 2021-09-29 at 13 03 02" src="https://user-images.githubusercontent.com/8155004/135256810-8b7e26a3-11ed-4d57-a1d1-805a69d15b9b.png">

<img width="486" alt="Screenshot 2021-09-29 at 13 03 12" src="https://user-images.githubusercontent.com/8155004/135256813-0f00b7fa-a861-4e8c-b319-bdf8277f3e3b.png">
